app.css

```diff
@@ -103,3 +103,753 @@ p {
   align-items: center;
   margin: 0% 0% 0% 7%;
 }
+
+.works-section {
+  height: auto;
+  background-color: #e5e5e5;
```

Comment:
- [ ] Currently, the `works-section` has the wrong background color applied. Kindly refer to the Figma design and ensure that you have applied the right background color to the section :100: 

| Your Design | The Figma Design |
|--------|--------|
| ![mobile](https://user-images.githubusercontent.com/102699958/227731876-25c7b9af-5934-4851-aa1f-86a28a0b32c0.png) | ![image](https://user-images.githubusercontent.com/102699958/227731845-08745193-5e51-4d42-ad39-6ff367176cb9.png) | 

- [ ] From the image below, you can see that there is a black color in the `works-section` on the right side and I think this is a result of the fixed with you applied to the section so please fix this as well so the entire section has only the color its supposed to have :100: 
- Your Design
![Screenshot from 2023-03-25 17-39-15](https://user-images.githubusercontent.com/102699958/227733107-67779369-0c05-4a68-aac3-c1aaf21987cf.png)